src/hotspot/share/prims/jvmtiEnvThreadState.cpp

```diff
@@ -335,6 +296,25 @@ class GetCurrentLocationClosure : public HandshakeClosure {
     }
     _completed = true;
   }
+  void do_vthread(Handle target_h) {
+    if (_target_jt != nullptr) {
+      assert(_target_jt->vthread() == target_h(), "sanity check");
+      do_thread(_target_jt);
+      return;
+    }
+    if (!JvmtiEnvBase::is_vthread_alive(target_h())) {
+      return; // _completed remains false.
+    }
```

Comment:
Do we need this? We already do this check in JvmtiHandshake::execute().

Reply:
Good suggestion, thanks. Will remove it.